util/travisScript.bash

```diff
@@ -24,10 +24,12 @@ makeCheckAndMason () {
 # Parses the last merge commit, getting the most recent package added to the registry
 checkPackage () {
   cd ..
-  package=$(git log -m -1 --name-only --pretty="format:")
+  package=$(git diff --name-only HEAD HEAD~1)
   end=".end"
   path="$package$end"
   cd $(dirname $path)
+  echo $package
```

Comment:
Mind adding a prefix to these for improved readability?

e.g.

```sh
echo "package detected from git diff: ${package}"
echo "package path: ${path}"
```

Reply:
@ben-albrecht Done 👍